lib/muffin/base.rb

```diff
@@ -5,6 +5,8 @@
 
 module Muffin
   class Base
+    include ActiveModel::Conversion
+    include ActiveModel::Naming
```

Comment:
It was already included implicitly via `Validation`, but having it explicit is a good idea.